test/performance/performance-tests.sh

```diff
@@ -54,7 +54,7 @@ function update_knative() {
   local n=0
   until [ $n -ge 3 ]
   do
-    ko apply -Rf serving/config/core/ && kubectl apply -f serving/third_party/net-istio.yaml && break
+    ko apply -Rf config/core/ && kubectl apply -f third_party/net-istio.yaml && break
```

Comment:
You should remove `kubectl apply -f third_party/net-istio.yaml`. 
Because `install_istio` above installs net-istio.yaml https://github.com/knative/serving/blob/a4249c02c2d015c58449d69600f156c287c66b64/test/e2e-networking-library.sh#L53, which does not have `PeerAuthentication`.

__BUT__ the net-istio.yaml in `install_istio()` replaces the namespace so probably you want to adjust it. https://github.com/knative/serving/blob/a4249c02c2d015c58449d69600f156c287c66b64/test/e2e-networking-library.sh#L51

Reply:
Thanks for the pointers, it worked!